public class MultipleCatchBlock4
{
    public static void main(String[]args)
    {
        try
        {
            String s=null;
            System.out.println(s.length());
        }
        catch(ArithmeticException e)
        {
            System.out.println("Arithmetic Exception occur");
        }
        catch(ArrayIndexOutOfBoundsException e)
        {
            System.out.println("Array index out of bouunds exception occur");
        }
        catch(Exception e)
        {
            System.out.println("Parent exception occur");
        }
        System.out.println("Rest of the code");
    }
}
